tcms/testruns/templates/testruns/get.html

```diff
@@ -115,6 +115,18 @@ <h2 class="card-pf-title" style="text-align: left">
                 <h2 class="card-pf-title">
                     {% trans 'Test executions' %}
                 </h2>
+                <div class="card-pf-view">
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-primary btn-lg">Add case</button>
+                        <button type="button" class="btn btn-primary btn-lg">Remove case</button>
+                        <button type="button" class="btn btn-primary btn-lg">Update case text</button>
+                        <button type="button" class="btn btn-primary btn-lg">Change assignee</button>
+                    </div>
+                    {% comment %} TODO: dropdown or modal - https://www.patternfly.org/v3/pattern-library/widgets/#modal {% endcomment %}
+                    <button type="submit" class="btn btn-primary btn-lg">{% trans "Change status" %}</button>
+                    <button type="submit" class="btn btn-primary btn-lg">{% trans "Add comment" %}</button>
+                    <button type="submit" class="btn btn-primary btn-lg">{% trans "Add hyperlink" %}</button>
```

Comment:
Buttons are too big. Use regular size.

Need to become a toolbar which will contain filter and actions, see:
https://www.patternfly.org/v3/pattern-library/forms-and-controls/actions/#design

Examples can be found at:
https://www.patternfly.org/v3/pattern-library/forms-and-controls/toolbar/#code

Reply:
I've copied the whole toolbar and applied our button onto it. 

This can serve as a baseline from which we can remove the filters/buttons we won't be using.